utils/from_mommy_to_bakery.py

```diff
@@ -62,11 +62,21 @@ def _replace_legacy_terms(file_path, dry_run):
             print(file_path)
 
 
+def _sanitize_folder_or_file(folder_or_file):
+    folder_or_file = folder_or_file.strip()
+    if folder_or_file.endswith('/'):
+        # Remove trailing slash e.g.: '.tox/' -> '.tox'
+        folder_or_file = folder_or_file[:-1]
+    return folder_or_file
+
+
 def check_files(dry_run):
     excluded_by_gitignore = [
-        folder_or_file.strip()
+        _sanitize_folder_or_file(folder_or_file.strip())
```

Comment:
You're already _stripping_ this in `_sanitize_folder_or_file`, so we can remove `.strip()`from here.

Reply:
Good catch, thanks for the review :bowing_man: